plat/hisilicon/poplar/aarch64/plat_helpers.S

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arch.h>
+#include <asm_macros.S>
+#include <bl_common.h>
+#include <cortex_a53.h>
+#include <cpu_macros.S>
+#include "hi3798cv200.h"
+#include "platform_def.h"
+
+	.globl	plat_get_my_entrypoint
+
+func plat_get_my_entrypoint
```

Comment:
I don't think this is needed as the secondary CPUs have boot address set to REG_PERI_CPU_RVBARADDR and so the PROGRAMMABLE_RESET_ADDRESS option should be used.